app/schedulers/background_scheduler.py

```diff
@@ -17,3 +26,14 @@ def init_scheduler():
     #                   replace_existing=True)
 
     scheduler.start()
+
+
+def init_delete_unverified_users_scheduler():
+    scheduler = BackgroundScheduler()
```

Comment:
Please do not create new scheduler just add job to existing one. Thanks in advance, this is the last change before we merge.

Reply:
apparently I didn't think about it😅Done, shall I squash the commits?

Reply:
yes please slash them :) Thank a lot!